pod/perlgit.pod

```diff
@@ -734,6 +734,32 @@ When describing the merge commit, explain the purpose of the branch, and
 keep in mind that this description will probably be used by the
 eventual release engineer when reviewing the next perldelta document.
 
+=head3 Merging via the GitHub.com interface
+
+The instructions above are written on the assumption that you are working from
+your the command line in your local git checkout and are pushing to GitHub.
+It is also possible to merge branches into blead (or into branches themselves
+branched from blead) via the GitHub.com interface.
+
+We have, however, noticed one peculiarity in this approach.  If the branch you
+are merging into another has multiple commits, and if you select the "squash
```

Comment:
In the case that I highlighted this problem occurring in (https://github.com/Perl/perl5/pull/18511), the source branch only had one commit so this is not specifically a problem to do with squashing multiple commits. I only used squash and merge because it was the default and I didn't think it would behave any differently from rebase and merge in this case.